README.md

```diff
@@ -90,6 +86,7 @@ The Only Headless CMS with a Visual Editor.
 
 #### Community
 
+- [Field Plugin Examples](https://github.com/storyblok/field-type-examples) Our community repository to collect different field-types to expand the Storyblok App
```

Comment:
not according [guidelines](https://github.com/storyblok/awesome-storyblok/blob/master/CONTRIBUTING.md).

> Use the following format: `[Title](link) - Description.`